providers/emailjs/src/lib/emailjs.provider.ts

```diff
@@ -31,15 +31,15 @@ export class EmailJsProvider implements IEmailProvider {
     html,
     attachments,
   }: IEmailOptions): Promise<ISendMessageSuccessResponse> {
-    const attachmentsModel: MessageAttachment[] = attachments?.map(
+    const attachmentsModel: MessageAttachment[] = attachments ? attachments.map(
       (attachment) => {
         return {
           name: attachment.name,
           data: attachment.file.toString(),
           type: attachment.mime,
         };
       }
-    );
+    ) : [];
 
     attachmentsModel?.push({ data: html, alternative: true });
```

Comment:
Should have even removed this check here. Completely unnecessary. Next PR

Reply:
The conditional operator? Yeah agree